plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/managers/CodeModernizerBottomWindowPanelManager.kt

```diff
@@ -238,20 +230,20 @@ class CodeModernizerBottomWindowPanelManager(private val project: Project) : JPa
         return actionManager.createActionToolbar(ACTION_PLACE, group, false)
     }
 
-    fun handleJobTransition(new: TransformationStatus, plan: TransformationPlan?, sourceJavaVersion: JavaSdkVersion) = invokeLater {
+    fun handleJobTransition(new: TransformationStatus, plan: TransformationPlan?, sourceJdk: JavaSdkVersion, transformType: CodeTransformType) = invokeLater {
         if (new in listOf(
                 TransformationStatus.PLANNED,
                 TransformationStatus.TRANSFORMING,
                 TransformationStatus.TRANSFORMED,
                 TransformationStatus.PAUSED,
                 TransformationStatus.COMPLETED,
                 TransformationStatus.PARTIALLY_COMPLETED
-            )
+            ) && transformType != CodeTransformType.SQL_CONVERSION // no plan for SQL conversions
```

Comment:
A bit unclear on this. Even if there is no plan for SQL conversions, how do we handle job transitions between the other states (e.g `TRANSFORMING` -> `PARTIALLY_COMPLETED`)

Reply:
Our polling function will stop polling when the job becomes PARTIALLY_COMPLETED or COMPLETED: https://github.com/aws/aws-toolkit-jetbrains/blob/fcb4f8814035730df2d96723fbf974aef122ef96/plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerSession.kt#L435

This part here simply prevents the (non-existent) plan from trying to open